lhotse/cut/mixed.py

```diff
@@ -38,31 +39,36 @@
 @dataclass
 class MixTrack:
     """
-    Represents a single track in a mix of Cuts. Points to a specific MonoCut and holds information on
+    Represents a single track in a mix of Cuts. Points to a specific MonoCut or MultiCut and holds information on
     how to mix it with other Cuts, relative to the first track in a mix.
     """
 
-    cut: Union[MonoCut, PaddingCut]
+    cut: Union[MonoCut, MultiCut, PaddingCut]
+    type: str = None
     offset: Seconds = 0.0
     snr: Optional[Decibels] = None
 
+    def __post_init__(self):
+        self.type = type(self.cut).__name__
+
     @staticmethod
     def from_dict(data: dict):
         raw_cut = data.pop("cut")
+        raw_cut_type = eval(data.pop("type"))
```

Comment:
in `CutSet.from_dicts` there's an close called `deserialize_one` that can disambiguate which type of cut to load. Can you promote it to a standalone function (or create `Cut.from_dict()` with that so that it just instantiates the right child class)? I would rather avoid an `eval()` here because what if somebody sends you a manifest with `MixTrack(type="import os; os.system('sudo rm -rf /')")`? 😀

Reply:
Yeah will do.